src/commons/index.js

```diff
@@ -27,5 +27,11 @@ module.exports = {
   },
   get withScrollReached() {
     return require('./withScrollReached').default;
+  },
```

Comment:
Let's extract the whole Modifiers file..

Reply:
Fixed